src/pages/workspace/WorkspaceResetBankAccountModal.js

```diff
@@ -1,45 +1,53 @@
 import lodashGet from 'lodash/get';
 import React from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
 import ConfirmModal from '../../components/ConfirmModal';
 import * as BankAccounts from '../../libs/actions/BankAccounts';
-import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 import * as ReimbursementAccountProps from '../ReimbursementAccount/reimbursementAccountPropTypes';
 import Text from '../../components/Text';
 import styles from '../../styles/styles';
 import BankAccount from '../../libs/models/BankAccount';
+import ONYXKEYS from '../../ONYXKEYS';
+import useLocalize from '../../hooks/useLocalize';
 
 const propTypes = {
     /** Reimbursement account data */
     reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
 
-    ...withLocalizePropTypes,
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }).isRequired,
 };
 
 function WorkspaceResetBankAccountModal(props) {
```

Comment:
Destructure props here
```suggestion
function WorkspaceResetBankAccountModal({reimbursementAccount, translate}) {
```

Reply:
Props have been destructured to `{reimbursementAccount, session}`